shell/platform/windows/fixtures/main.dart

```diff
@@ -86,6 +87,42 @@ void alertPlatformChannel() async {
   ui.PlatformDispatcher.instance.sendPlatformMessage('flutter/accessibility', byteData, (ByteData? _){});
 }
 
+@pragma('vm:entry-point')
+void exitTestExit() async {
+  final Completer<ByteData?> closed = Completer<ByteData?>();
+  ui.channelBuffers.setListener('flutter/platform', (ByteData? data, ui.PlatformMessageResponseCallback callback) async {
+    const String jsonString = '[{"response":"exit"}]';
```

Comment:
Optional: rather than hand-writing JSON, consider doing something like:
```dart
final String jsonString = json.encode(<String, dynamic>{"response": "exit"});
```

Mostly because that allows the compiler to catch Dart syntax errors and json.encode to catch JSON syntax errors.